packages/nextjs/components/Header.tsx

```diff
@@ -4,14 +4,28 @@ import Link from "next/link";
 import Image from "next/image";
 import { Faucet } from "~~/components/scaffold-eth";
 import RainbowKitCustomConnectButton from "~~/components/scaffold-eth/RainbowKitCustomConnectButton";
-import { BugAntIcon, SparklesIcon } from "@heroicons/react/24/outline";
+import { BugAntIcon, SparklesIcon, HomeModernIcon, BeakerIcon } from "@heroicons/react/24/outline";
+
+function checkAndCloseDropDown(e: { currentTarget: any }) {
+  const targetEl = e.currentTarget;
+  if (targetEl && targetEl.matches(":focus")) {
+    setTimeout(function () {
+      targetEl.blur();
+    }, 0);
+  }
+}
 
 const NavLink = ({ href, children }: { href: string; children: React.ReactNode }) => {
   const router = useRouter();
 
   return (
     <Link href={href} passHref>
       <a
+        onClick={() => {
+          if (document.activeElement instanceof HTMLElement) {
+            document.activeElement.blur();
+          }
+        }}
```

Comment:
We won't need this either if we use state.

Reply:
I've left this in atm b/c it works when separating Navlinks into their own component and doesn't break anything in my testing.